open_lm/params.py

```diff
@@ -90,6 +90,8 @@ def parse_args(args):
     )
     parser.add_argument(
         "--dataset-metadata",
+        type=str,
```

Comment:
nit: dataset manifest instead of metadata?

Reply:
Changed this into dataset-manifest.